tools/changelog.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Grab the changelog for issues."""
+
+import datetime
+import os
+import sys
+from urllib import quote_plus
+from urlparse import urljoin
+
+import requests
+
+sys.path.append(os.path.realpath(os.pardir))
+from config.secrets import OAUTH_TOKEN
+
+
+# Config
+
+URL_REPO = 'https://api.github.com/repos/webcompat/webcompat.com/issues'
+SEARCH_ARG = '?labels='
+SEARCH_STRING = 'status: add to changelog'
+LINE_TEMPLATE = '* {title} [Pull #{number}]({url})\n'
+LOG_TEMPLATE = """X.X.X - {date}
+{loglines}
+"""
+
+
+def get_remote_file(url):
+    """Request URL."""
+    headers = {'Accept': 'application/vnd.github.v3+json',
+               'Authorization': 'token {token}'.format(token=OAUTH_TOKEN),
+               'User-Agent': 'webcompat/webcompat-bot'}
+    r = requests.get(url, headers=headers)
+    json_response = r.json()
+    return json_response
+
+
+def create_changelog(json_response):
+    """Extract the number of open issues."""
+    loglines = ''
+    for issue in json_response:
+        title = issue['title']
+        number = issue['number']
+        url = issue['url']
+        loglines += LINE_TEMPLATE.format(title=title, url=url, number=number)
+        status = delete_label(number, SEARCH_STRING)
```

Comment:
Maybe this is a feature request/follow up bug:

It would be cool if it spit out the changelog, then prompted you Y/N to delete the labels. That way you could run it more than once (for whatever reason?). As I write this down I'm not sure this is useful, so take it with a grain of salt.

Reply:
Random argument in favor of making deletion intentional via a y/n prompt: if someone accidentally runs the script, it's non-destructive.

Reply:
yup that's a good idea. I will work on your comments later on this week.